### PR TITLE
Cvsl 2256 afer com residential checks show com caseload

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: eslint.config.mjs [hunk 1]
```diff
@@ -3,7 +3,6 @@ import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
 export default hmppsConfig({
   extraIgnorePaths: ['test_results'],
   extraPathsAllowingDevDependencies: ['playwright.config.ts'],
```
Comment (nishanthmahasamudram) [inline eslint.config.mjs:5]: extraGlobals is no longer needed.

File: server/app.ts [hunk 2]
```diff
@@ -45,8 +45,8 @@ export default function createApp(services: Services, applicationInfo: Applicati
   app.use(setUpAuthentication())
   app.use(authorisationMiddleware())
   app.use(setUpCsrf())
```
Comment (nishanthmahasamudram) [inline server/app.ts:47]: moved populating current user details below populating client token as we need client token to fetch staff details in order to populate staff details like id, code, etc.

File: server/routes/communityOffenderManager/caseload.ts [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
   GET = async (req: Request, res: Response): Promise<void> => {
-    res.render('pages/communityOffenderManager/caseload')
+    const { user } = res.locals
+    const offenderSummaryList = await this.communityOffenderManagerCaseloadService.getCommunityOffenderManagerCaseload(
+      req?.middleware?.clientToken,
+      user as ProbationUser,
+    )
+
+    const caseload = offenderSummaryList.map(offender => {
+      return {
```
Comment (nishanthmahasamudram) [inline server/routes/communityOffenderManager/caseload.ts:16]: will have to update create link based on the next loading page.

File: server/routes/communityOffenderManager/caseload.test.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+describe('GET', () => {
+  it('should render list of licences for approval', async () => {
+    await caseloadRoutes.GET(req, res)
+    expect(communityOffenderManagerCaseloadService.getCommunityOffenderManagerCaseload).toHaveBeenCalledWith(
+      req.middleware.clientToken,
+      res.locals.user as ProbationUser,
+    )
+    expect(res.render).toHaveBeenCalledWith('pages/communityOffenderManager/caseload', {
+      caseload: [
+        {
```
Comment (nishanthmahasamudram) [inline server/routes/communityOffenderManager/caseload.test.ts:41]: will have to update create link based on the next loading page.

File: server/app.ts (hunk 2 above)
Comment (andrewrlee) [reply server/app.ts:47]: but will the username exist on the request.user object? to send through when requesting the client token?

File: server/routes/communityOffenderManager/caseload.test.ts (hunk 4 above)
Comment (andrewrlee) [reply server/routes/communityOffenderManager/caseload.test.ts:41]: **nit:** might want to call it the reviewLink or something

File: server/app.ts (hunk 2 above)
Comment (nishanthmahasamudram) [reply server/app.ts:47]: **answer**: Yes, it is set [here](https://github.com/ministryofjustice/hmpps-assess-for-early-release-ui/blob/31d0be8f9ee6cb0933600827cfdf5123b6a29d22/server/middleware/setUpAuthentication.ts#L55) before that.

**Pull request merged**